### PR TITLE
Add darwin arm64 target

### LINKED ISSUE #261: No provider available for running on apple silicon
Hi,

When trying to use your provider I get the following error:

```
Provider registry.terraform.io/rollbar/rollbar v1.4.0 does not have a package available for your current platform, darwin_arm64.
```

This means that I'm unable to use the provider when running Terraform locally. Can you please add a release for `darwin_arm64` for v1.4.0?

### PR DESCRIPTION
**Pull request by jhottenstein**
## Description of the change

Adds arm64 as an architecture for darwin goos

This will make it possible for someone on macos silicon to use this provider

## Type of change

- [ ] New feature (non-breaking change that adds functionality)

## Related issues

Resolves #261 

- Fix #261

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

**Comment by jhottenstein** [comment]
Hello,

I don't know much about go or terraform plugin development.  Is this sufficient to make a darwin arm64 binary so that I can use your plugin?  Happy to modify if I need to do more.

Thanks!

File: Makefile [hunk 1]
```diff
@@ -36,12 +37,12 @@ install012: build
...
-test: 
+test:
 	go test -covermode=atomic -coverprofile=coverage.out $(TEST) || exit 1
-	@#echo $(TEST) | xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4                    
+	@#echo $(TEST) | xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
 
-testacc: 
-	TF_ACC=1 TERRAFORM_PROVIDER_ROLLBAR_DEBUG=1 go test -covermode=atomic -coverprofile=coverage.out $(TEST) -v $(TESTARGS) -timeout 120m   
+testacc:
+	TF_ACC=1 TERRAFORM_PROVIDER_ROLLBAR_DEBUG=1 go test -covermode=atomic -coverprofile=coverage.out $(TEST) -v $(TESTARGS) -timeout 120m
```
Comment (jhottenstein) [inline Makefile:45]: My IDE cleaned up this whitespace.  Happy to omit.

**Comment by haggishunk** [comment]
Hi @jhottenstein, building for `darwin_arm64` will require golang 1.16 at a minimum.  Appreciate the effort in making this provider more usable on different platforms!

File: Makefile (hunk 1 above)
Comment (haggishunk) [reply Makefile:45]: This is ok.

**Comment by jhottenstein** [comment]
Hello!  Thanks for the reply.  I've updated the go version in the module.  Anything else?

**Comment by kyledecot** [comment]
🎉 Thanks for making these changes @jhottenstein

**Comment by jhottenstein** [comment]
@haggishunk Do I need to do anything?  Thanks!

**Review by haggishunk — APPROVED** [review]
@jhottenstein Thank you for taking the time to make this PR.  Apologies for not getting back to you earlier on this... Rollbar is taking a renewed interest in making sure the terraform provider is actively maintained and moderated.

**Comment by KieranP** [comment]
@haggishunk Are we able to get a release for darwin_arm64? Running `terraform init -upgrade`, I'm still getting:

```
Provider registry.terraform.io/rollbar/rollbar v1.5.0 does not have a package available for your current platform, darwin_arm64.
```

As such, I've been unable to run my terraform scripts for weeks since changing to a new M1 Mac.

**Comment by cyrusradfar** [comment]
Thanks for the suggestions. Product team will look into prioritizing the update.

**Pull request merged**